src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
```

Comment:
Remove the min-width property to allow the layout to be responsive for screens smaller than 1024px.

Reply:
done